### PR TITLE
docs(force-https): explain HSTS override

### LINKED ISSUE #2730: Document Force HTTPS/Strict-Transport-Security behaviour
Following https://github.com/Scalingo/scalingo-openresty/issues/128 the behaviour of the Force HTTPS button evolves, the user can send back the Strict-Transport-Security and we won't override it.

We should document the possible values of the header, and what the consequences are

### PR DESCRIPTION
**Pull request by EtienneM**
Fix #2730

### CODE REVIEW

File: src/_posts/platform/app/2000-01-01-force-https.md [hunk 1]
```diff
@@ -23,14 +23,14 @@ Or, once a custom domain has been added:
...
+redirection from HTTP to HTTPS and injecting the `Strict-Transport-Security` header (abbreviated HSTS) in HTTPS responses.
 
 ## Force HTTPS a Scalingo Application
 
 Scalingo eases the process for you to enable "Force HTTPS" on your application.
-Head to the Settings tab of your web dashboard and check "Force HTTPS":
+Head to the Settings tab of your web dashboard, in the Routing section, check "Force HTTPS":
 
-{% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_dashboard_settings.png" %}
+{% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_dashboard_force_https.png" %}
```
Comment (EtienneM) [inline src/_posts/platform/app/2000-01-01-force-https.md:33]: I updated the screenshot in the CDN. After this PR merge, I will remove screenshot_dashboard_settings.png from the CDN.

File: src/_posts/platform/app/2000-01-01-force-https.md [hunk 2]
```diff
@@ -48,29 +48,27 @@ using the status code `301`. It means browsers will remember this redirection
 over time:
 
 ```console
-$ curl http://my-app.osc-fr1.scalingo.io -I
+$ curl --head http://my-app.osc-fr1.scalingo.io
```
Comment (EtienneM) [inline src/_posts/platform/app/2000-01-01-force-https.md:51]: Long options are better in documentation pages

File: src/_posts/platform/app/2000-01-01-force-https.md [hunk 3]
```diff
@@ -107,16 +107,18 @@ Network](https://developer.mozilla.org/docs/S%C3%A9curit%C3%A9/HTTP_Strict_Trans
...
+    > If the 301 status code is received in response to a request other
+    > than GET or HEAD, the user agent MUST NOT automatically redirect the
+    > request unless it can be confirmed by the user, since this might
+    > change the conditions under which the request was issued.
+    >
+    > Note: When automatically redirecting a POST request after
+    > receiving a 301 status code, some existing HTTP/1.0 user agents
+    > will erroneously change it into a GET request.
+
+    (from [ietf.org/rfc/rfc2616.txt](https://www.ietf.org/rfc/rfc2616.txt) section 10.3.2)
```
Comment (EtienneM) [inline src/_posts/platform/app/2000-01-01-force-https.md:122]: This block didn't rendered well. I fixed it.

**Review by mcimbaro — APPROVED** [review]
Nice work!

**Pull request merged**